src/frontend/src/catalog/system_catalog/pg_catalog/pg_settings.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright 2022 Singularity Data
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::LazyLock;
+
+use risingwave_common::row::Row;
+use risingwave_common::types::DataType;
+
+use crate::catalog::system_catalog::SystemCatalogColumnsDef;
+
+/// The catalog `pg_settings` stores settings.
+/// Ref: [`https://www.postgresql.org/docs/current/catalog-pg-namespace.html`]
+pub const PG_SETTINGS_TABLE_NAME: &str = "pg_settings";
+pub const PG_SETTINGS_COLUMNS: &[SystemCatalogColumnsDef<'_>] =
+    &[(DataType::Varchar, "name"), (DataType::Varchar, "setting")];
+
+pub static PG_SETTINGS_DATA_ROWS: LazyLock<Vec<Row>> =
+    LazyLock::new(|| vec![Row::new(vec![None, None])]);
```

Comment:
Seems like it is not stright-forward to use same rows from `SHOW ALL` since we don't have the session configs in place. I will keep 0 row for now and may refactor it in the future.